templates/firefox-ios.xliff

```diff
@@ -1319,6 +1344,102 @@
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.DownloadLink" xml:space="preserve">
+        <source>Download Link</source>
+        <note>A label indicating the keyboard shortcut of downloading a link the user taps on. This label is displayed in the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.FindAgain" xml:space="preserve">
+        <source>Find Again</source>
+        <note>A label indicating the keyboard shortcut of finding text a user desires within a page again. This label is displayed in the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.OpenLinkInBackground" xml:space="preserve">
+        <source>Open Link in Background</source>
+        <note>A label indicating the keyboard shortcut of opening a link in a new tab while staying on the currently selected tab. This label is displayed in the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.OpenLinkInNewTab" xml:space="preserve">
+        <source>Open Link in New Tab</source>
+        <note>A label indicating the keyboard shortcut of opening a link in a new tab and switching to that tab at the same time. This label is displayed in the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.SavePageAs" xml:space="preserve">
+        <source>Save Page As...</source>
+        <note>A label indicating the keyboard shortcut of saving the current web page in a format of the user's choice. This label is displayed in the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.Section.Bookmark" xml:space="preserve">
+        <source>Bookmarks</source>
+        <note>A label indicating a grouping of related keyboard shortcuts describing actions a user can do with Bookmarks. This label is displayed inside the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.Section.Edit" xml:space="preserve">
+        <source>Edit</source>
+        <note>A label indicating a grouping of related keyboard shortcuts describing actions a user can do within a web page. This label is displayed inside the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.Section.File" xml:space="preserve">
+        <source>File</source>
+        <note>A label indicating a grouping of related keyboard shortcuts describing actions a user can take on, and within, a Tab. This label is displayed inside the Discoverability overlay when a user presses the Command key.</note>
```

Comment:
Is the comment relevant to the string "File"?
For some reason, I can't relate it to the string

Reply:
According to design, certain actions should be grouped. So although these don't really relate directly to the string, they'll still exist under that category. You can have a look at this Figma as well: 

https://www.figma.com/file/4Vlr0idCIGQE3r9I1FnkRn/Keyboard-Shortcuts?node-id=2%3A7